packages/matchbox/src/components/Button/Button.js

```diff
@@ -111,7 +111,7 @@ const Button = React.forwardRef(function Button(props, ref) {
 
   const loadingIndicator = React.useMemo(() => {
     return (
-      <Transition mountOnEnter unmountOnExit in={loading}>
+      <Transition mountOnEnter unmountOnExit in={loading} timeout="500">
```

Comment:
I think this needs to be a number: https://reactcommunity.org/react-transition-group/transition#Transition-prop-timeout

And we should also just use 0 here for no timeout